src/components/CardItem/CardItem.test.js

```diff
@@ -10,14 +13,46 @@ function renderCardItem(args) {
     cardProp: {
       id: 0,
       text: '',
-      userName: ''
+      username: '',
+      votes: 0
     }
   };
   const props = { ...defaultProps, ...args };
   return render(<CardItem {...props} />);
 }
 
 it('shows the user name on the card', () => {
-  const { getByText } = renderCardItem({ cardProp: { userName: 'John Doe' } });
+  const { getByText } = renderCardItem({ cardProp: { username: 'John Doe' } });
   getByText('John Doe');
 });
+
+it('shows a number for the amount of votes', () => {
+  const { getByText } = renderCardItem();
+  getByText('0');
+});
+
+it('has an up-vote button', () => {
+  const { getByRole } = renderCardItem();
+  getByRole('button');
+});
+
+it('increments the counter when you click up-vote', async () => {
+  await act(async () => {
+    const { getByRole, getByText } = renderCardItem();
+    getByRole('button').click();
+    await waitFor(() => {
+      getByText('1');
```

Comment:
If I change the number from 1 to 0, the test still passes. I guess the test is not testing the correct thing. Unfortunately, I don't really have any suggestions or ideas about why is that the case. 

If you don't fix it please remove the test.

Reply:
if you change it to 2, does it also pass? if you wait for it to change to 0 it should pass because it starts at 0

Reply:
No it does not pass with 2.